File: semestr-8/Programowanie obiektowe w Java/alexander/POJ/2023s/IZ08IO2/StrategyProject/src/SportsmenCommandMultiply.java
public class SportsmenCommandMultiply implements SportsmenCommand
{
    private String title;
    private SportsmenCommand command;
    private int count;

    public SportsmenCommandMultiply(String title, SportsmenCommand command, int count)
    {
        this.title = title;
        this.command = command;
        this.count = count;
    }
        
    @Override public String execute(Sportsmen sportsmen)
    {
        return String.format("%d x %s", count, command.execute(sportsmen));
    }
    
    @Override public String toString() { return title; }
}

File: semestr-8/Jezyk C#/alexander/Csharp/Archiwum/2022s/ID06TC1/Command/Command/SportsmenCommandMultiply.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Command
{
    internal class SportsmenCommandMultiply: SportsmenCommandComplex
    {
        private uint count;
        private SportsmenCommand command;
        public SportsmenCommandMultiply(string name, uint count, SportsmenCommand command) : base(name)
        {
            this.count = count;
            this.command = command;
        }

        public override string execute(Sportsmen sportsmen)
        {
            return count.ToString() + " * " + command.execute(sportsmen);
        }
    }
}
